### ISSUE TITLE
Add EPEL distros to tarantool.repo (CentOS/Amazon Linux)

### ISSUE THREAD

**Issue by kostja** [tarantool/tarantool#1331]
Most tarantool packages for CentOS on tarantool.org/dist depend on packages in EPEL repository. Yet enabling Tarantol upstream on CentOS, using the instructions on our web site, does not enable EPEL. This leads to installation failures if EPEL was not enabled before. Either mirror all relevant packages (libyaml, python-msgpack, perl-anyevent, etc, etc, etc) in our repo, or change download instructions to add EPEL as upstream (preferrable with a single copy and paste).

**Comment by rtsisyk** [comment]
tarantool package from tarantool.org shouldn't depend on EPEL packages.
I'll check CentOS 6 / CentOS 7.

**Comment by rtsisyk** [comment]
CentOS 6 OK:

```
[root@03a4fe3ec98a 1.6]# yum repolist
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror.h1host.ru
repo id                                                                                                        repo name                                                                                                                 status
base   <!-- Only Base repo is enabled                                                                                                        CentOS-6 - Base                                                                                                           6,575  
tarantool                                                                                                      CentOS-6 - Tarantool                                                                                                        615
repolist: 7,190
[root@03a4fe3ec98a 1.6]# yum install tarantool
Loaded plugins: fastestmirror
Setting up Install Process
Loading mirror speeds from cached hostfile
 * base: mirror.h1host.ru
Resolving Dependencies
--> Running transaction check
---> Package tarantool.x86_64 0:1.6.8.487-1.el6 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

===============================================================================================================================================================================================================================================
 Package                                                  Arch                                                  Version                                                         Repository                                                Size
===============================================================================================================================================================================================================================================
Installing:
 tarantool                                                x86_64                                                1.6.8.487-1.el6                                                 tarantool                                                1.2 M

Transaction Summary
===============================================================================================================================================================================================================================================
Install       1 Package(s)

```

**Comment by rtsisyk** [comment]
A clean CentOS 6 from Docker Hub:
```
===============================================================================================================================================================================================================================================
 Package                                                       Arch                                               Version                                                          Repository                                             Size
===============================================================================================================================================================================================================================================
Installing:
 tarantool                                                     x86_64                                             1.6.8.487-1.el6                                                  tarantool                                             1.2 M
Installing for dependencies:
 hwdata                                                        noarch                                             0.233-14.1.el6                                                   base                                                  1.3 M
 initscripts                                                   x86_64                                             9.03.49-1.el6.centos                                             base                                                  945 k
 iproute                                                       x86_64                                             2.6.32-45.el6                                                    base                                                  367 k
 iptables                                                      x86_64                                             1.4.7-16.el6                                                     base                                                  254 k
 iputils                                                       x86_64                                             20071127-20.el6                                                  base                                                  120 k
 libgomp                                                       x86_64                                             4.4.7-16.el6                                                     base                                                  134 k
 libsemanage                                                   x86_64                                             2.0.43-5.1.el6                                                   base                                                  104 k
 libyaml                                                       x86_64                                             0.1.3-4.el6_6                                                    base                                                   52 k
 mingetty                                                      x86_64                                             1.08-5.el6                                                       base                                                   21 k
 module-init-tools                                             x86_64                                             3.9-25.el6                                                       base                                                  466 k
 policycoreutils                                               x86_64                                             2.0.83-24.el6                                                    base                                                  651 k
 psmisc                                                        x86_64                                             22.6-19.el6_5                                                    base                                                   81 k
 sysvinit-tools                                                x86_64                                             2.87-6.dsf.el6                                                   base                                                   60 k
 udev                                                          x86_64                                             147-2.63.el6                                                     base                                                  355 k
 upstart                                                       x86_64                                             0.6.5-13.el6_5.3                                                 base                                                  177 k
 util-linux-ng                                                 x86_64                                             2.17.2-12.18.el6                                                 base                                                  1.6 M

Transaction Summary
===============================================================================================================================================================================================================================================
Install      17 Package(s)

Total download size: 7.8 M
Installed size: 29 M

Installed:
  tarantool.x86_64 0:1.6.8.487-1.el6                                                                                                                                                                                                           

Dependency Installed:
  hwdata.noarch 0:0.233-14.1.el6           initscripts.x86_64 0:9.03.49-1.el6.centos   iproute.x86_64 0:2.6.32-45.el6      iptables.x86_64 0:1.4.7-16.el6            iputils.x86_64 0:20071127-20.el6         libgomp.x86_64 0:4.4.7-16.el6  
  libsemanage.x86_64 0:2.0.43-5.1.el6      libyaml.x86_64 0:0.1.3-4.el6_6              mingetty.x86_64 0:1.08-5.el6        module-init-tools.x86_64 0:3.9-25.el6     policycoreutils.x86_64 0:2.0.83-24.el6   psmisc.x86_64 0:22.6-19.el6_5  
  sysvinit-tools.x86_64 0:2.87-6.dsf.el6   udev.x86_64 0:147-2.63.el6                  upstart.x86_64 0:0.6.5-13.el6_5.3   util-linux-ng.x86_64 0:2.17.2-12.18.el6  

Complete!
[root@aa8bff707213 /]# tarantool
tarantool: version 1.6.8-487-g2d8b6cb
type 'help' for interactive help
tarantool> 
```

**Comment by rtsisyk** [comment]
A clean CentOS 7 from DockerHub:

```
[root@37c3c2093d41 /]# yum repolist
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: centos-mirror.rbc.ru
repo id                                                                                                            repo name                                                                                                             status
base/7/x86_64                                                                                                      CentOS-7 - Base                                                                                                       9007
tarantool/7/x86_64                                                                                                 CentOS-7 - Tarantool                                                                                                   954
repolist: 9961
[root@37c3c2093d41 /]# yum install tarantool
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: centos-mirror.rbc.ru
Resolving Dependencies
--> Running transaction check
---> Package tarantool.x86_64 0:1.6.8.487-1.el7.centos will be installed
--> Processing Dependency: libgomp.so.1(GOMP_2.0)(64bit) for package: tarantool-1.6.8.487-1.el7.centos.x86_64
--> Processing Dependency: libgomp.so.1(GOMP_1.0)(64bit) for package: tarantool-1.6.8.487-1.el7.centos.x86_64
--> Processing Dependency: libyaml-0.so.2()(64bit) for package: tarantool-1.6.8.487-1.el7.centos.x86_64
--> Processing Dependency: libgomp.so.1()(64bit) for package: tarantool-1.6.8.487-1.el7.centos.x86_64
--> Running transaction check
---> Package libgomp.x86_64 0:4.8.5-4.el7 will be installed
---> Package libyaml.x86_64 0:0.1.4-11.el7_0 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

===============================================================================================================================================================================================================================================
 Package                                                Arch                                                Version                                                               Repository                                              Size
===============================================================================================================================================================================================================================================
Installing:
 tarantool                                              x86_64                                              1.6.8.487-1.el7.centos                                                tarantool                                              1.2 M
Installing for dependencies:
 libgomp                                                x86_64                                              4.8.5-4.el7                                                           base                                                   130 k
 libyaml                                                x86_64                                              0.1.4-11.el7_0                                                        base                                                    55 k

Transaction Summary
===============================================================================================================================================================================================================================================
Install  1 Package (+2 Dependent packages)

Total download size: 1.4 M
Installed size: 3.6 M
Is this ok [y/d/N]: 

[CUT]                                                          

Installed:
  tarantool.x86_64 0:1.6.8.487-1.el7.centos                                                                                                                                          
Dependency Installed:
  libgomp.x86_64 0:4.8.5-4.el7                                                                                         libyaml.x86_64 0:0.1.4-11.el7_0                                                                                        

Complete!

[root@37c3c2093d41 /]# tarantool
tarantool: version 1.6.8-487-g2d8b6cb
type 'help' for interactive help
tarantool> 
```

**Comment by rtsisyk** [comment]
EPEL is needed for python-msgpack
Fixed via 12e35278b467f448d1dc4b331aaad254575a938e
